server/models/ChartsModel/index.js

```diff
@@ -4,15 +4,15 @@ import { ASC } from '../../constants'
 
 const id = '$_id'
 const idKey = '$chart_id'
-const chartId = 'chart_id'
 const updatedAt = 'updatedAt'
 const $updatedAt = '$updatedAt'
+const today = new Date()
```

Comment:
this needs to be defined in the functions themselves otherwise it will be evaluated once and all the timestamps will have that value.

Reply:
forgot the parens 
[aae6c0c](https://github.com/AMP-SCZ/dpdash/pull/704/commits/aae6c0cac85e6148b65069005a4a8cee515f5305)